pennychat/tests/api/test_penny_chat.py

```diff
@@ -38,13 +77,34 @@ def test_penny_chat_participants_list(test_chats_1):
         assert int(user_id) in [participant['user']['id'] for participant in chat['participants']]
```

Comment:
This test proves that if a chat is returned, then it is a chat that they participated in. However it doesn't prove the corollary, that if they participated in a chat it would be returned EVEN if it's private.

I would therefore add something like this:
```
returned_chat_ids = set([chat['id'] for chat in chats])   # see https://book.pythontips.com/en/latest/comprehensions.html to understand this syntax
assert chat_private_1.id in returned_chat_ids, 'user organized this chat but it was not returned'
assert chat_private_2.id in returned_chat_ids, 'user attended this chat but it was not returned'
```